data/decks/mika.json

```diff
@@ -0,0 +1,182 @@
+{
+	"id": "mika",
+	"season": 6,
+	"display_name": "Mika Returna",
+	"cardback": "cardback_mika.jpg",
+	"exceed_cost": 1,
+	"character_action_default": [{
+		"gauge_cost": 0,
+		"force_cost": 0,
+		"effect": {
+			"effect_type": "self_discard_choose",
+			"optional": false,
+			"amount": 1,
+			"and": {
```

Comment:
Does and work after this?
maybe use the discard_effect stuff I added.

Reply:
the "and" actually is supported by the character action handler, and I wanted to avoid discard_effect because of discarding 0 technically being legal (though as mentioned above we might not have to worry about that)